crt_portal/cts_forms/views.py

```diff
@@ -490,10 +489,10 @@ def get_context_data(self, **kwargs):
         choices = {
             key: {
                 'description': description,
-                'helptext': PRIMARY_COMPLAINT_CHOICES_TO_HELPTEXT.get(key, ''),
                 'examples': PRIMARY_COMPLAINT_CHOICES_TO_EXAMPLES.get(key, [])
             }
             for key, description in PRIMARY_COMPLAINT_DICT.items()
+            if key != 'something_else'
```

Comment:
I feel like I don't see multi-line comprehensions like this all that often, super readable, love it!

Reply:
A short docstring could be helpful here to describe why that `something_else` key is being exclude

Reply:
Thanks! And added comment. This was a last minute addition, agree it should have an explanation as to why.